conf/messages

```diff
@@ -272,6 +272,76 @@ tax-year-summary.message.C559105                                   = the amount
 tax-year-summary.message.C559113                                   = for your foreign property, you need to submit either consolidated or detailed expenses but not both.
 tax-year-summary.message.C559114                                   = for your EEA furnished holiday lettings, you need to submit either consolidated or detailed expenses but not both.
 
+tax-year-summary.agent.message.header                              = We cannot show this calculation because:
+tax-year-summary.agent.message.action                              = You need to amend and resubmit your client’s return
+tax-year-summary.agent.message.C15014                              = a quarterly update cannot end after the accounting period end date of {0}.
+tax-year-summary.agent.message.C55014                              = a quarterly update cannot end after the accounting period end date of {0}.
+tax-year-summary.agent.message.C15015                              = your client has claimed to carry forward a loss to set against general income of the next year. They also need to make the claim in the same year the loss arose.
+tax-year-summary.agent.message.C15016                              = your client uses cash basis accounting. This means that they cannot claim to set losses against other taxable income.
+tax-year-summary.agent.message.C15102                              = the total amount of one-off Gift Aid payments is more than the total Gift Aid payments your client made.
+tax-year-summary.agent.message.C15103                              = Gift Aid payments made this year treated as paid in the previous year are more than the total Gift Aid payments your client made.
+tax-year-summary.agent.message.C15104                              = the value of qualifying investments your client gifted to non-UK charities are more than the value of these items they gifted to charity:<br><ul class="govuk-list govuk-list--bullet"> <li> qualifying shares and securities </li> <li> qualifying land and buildings </li> </ul>
```

Comment:
this might be not a good idea to keep HTML formatting as a part of resource file, we have views for this

Reply:
It's been implemented for the individual so I have implemented the same for Agent

Reply:
no html and styles in the message files pls